timeseries/src/autogluon/timeseries/models/gluonts/abstract_gluonts.py

```diff
@@ -261,54 +260,49 @@ def _predict_gluonts_forecasts(self, data: TimeSeriesDataFrame, **kwargs) -> Lis
 
         return list(self.gts_predictor.predict(**predictor_kwargs))
 
+    @staticmethod
+    def _sample_to_quantile_forecast(forecast: np.ndarray, quantile_levels: List[float]) -> QuantileForecast:
+        forecast_arrays = []
```

Comment:
Is `forecast` a 1D array of sampled values at a certain timestamp? That is, we call `_sample_to_quantile_forecast` `data.num_items * prediction_length` times?

Maybe it makes sense to add a short annotation explaining the expected shapes of arrays?